native/src/routes/FeedbackModalContainer.tsx

```diff
@@ -11,7 +11,7 @@ type FeedbackModalContainerProps = {
 }
 
 const FeedbackModalContainer = ({ route }: FeedbackModalContainerProps): ReactElement => (
-  <FeedbackContainer {...route.params} />
+  <FeedbackContainer {...route.params} noResults={false} />
```

Comment:
noResults is optional you can just add `noResults = false`  at the FeedbackContainer props and remove this..
```suggestion
  <FeedbackContainer {...route.params} />
```